ApiBundle/Controller/BlockController.php

```diff
@@ -125,4 +125,27 @@ public function listSharedBlockByComponentAction($language, $component)
 
         return $this->get('open_orchestra_api.transformer_manager')->transform('block_collection', $collection);
     }
+
+    /**
+     * @param string  $blockId
+     *
+     * @Config\Route("/share/block/{blockId}", name="open_orchestra_api_block_share")
+     * @Config\Method({"PUT"})
+     *
+     * @return FacadeInterface
+     */
+    public function shareBlockAction($blockId)
+    {
+        $this->denyAccessUnlessGranted(ContributionActionInterface::CREATE, BlockInterface::ENTITY_TYPE);
```

Comment:
i'm not sur for role what is the role to create transverse block

Reply:
this one! (come from Admin/BlockController)